shell/src/dapps/o-marketlisting/pages/MarketlistingDetail.svelte

```diff
@@ -165,7 +173,7 @@ function checkIsOpenNow(timesArray) {
       if (times[1]) {
         let begIn = convertH2M(times[0]);
         let endIn = convertH2M(times[1]);
-        return (open = nIn >= begIn && (nIn < endIn || nIn === endIn));
+        return (open = nIn >= begIn && nIn < endIn);
```

Comment:
Please check if that is equivalent and if yes, change it to the suggestion.
```suggestion
        open = nIn >= begIn && nIn < endIn;
        return open;
```